Fix CHANGELOG.md

Djelibeybi opened the pull request:
The RPM changelog generation needs a date value for the 1.9.1 release. The date is probably incorrect though, because of timezones and such, so please adjust to the actual correct date you want to use for the release of 1.9.1. 

Signed-off-by: Avi Miller avi.miller@gmail.com

Djelibeybi committed:
Fix CHANGELOG.md so that RPM changelog generation has a date value for the 1.9.1 release.

Signed-off-by: Avi Miller <avi.miller@gmail.com>

Djelibeybi commented:
Ping @jfrazelle as this probably breaks the RPM changelog generation and should be cherry-picked back into 1.9.x

thaJeztah commented:
@Djelibeybi so, IIUC, this requires the RPMs to be rebuilt? Is that possible without creating a new release?

thaJeztah commented:
Also; @tiborvass @jfrazelle think this should be added to the release checklist

Djelibeybi commented:
I don't think it's bad enough to require a rebuild, but I do think it should be included in 1.9.2. And yes, the format of the CHANGELOG.md should be considered in the checklist.

thaJeztah commented:
@Djelibeybi thanks! I'm not sure a 1.9.2 release is planned (only if serious issues arise in the 1.9.1 release), but if a 1.9.2 release will be created, yes, we should cherry-pick.

LGTM, but don't merge yet, pending the discussion above!

Djelibeybi commented:
Note that I haven't actually checked your 1.9.1 RPM. I assume you do post-build install checks, but if not, I recommend verifying the RPM installs ok. I assume it does, otherwise there would've been lots more issues raised. :) 

We fixed this in our downstream build when we added our local changes.

calavera commented:
I think we should merge this into master and leave it there for now. I've created a 1.9.2 milestone to keep track of possible patches to release, like this one, but we won't release it unless we find any critical bug worth a new release.

LGTM

thaJeztah commented:
Thanks @calavera for having a look, and @Djelibeybi for reporting!
